priv/repo/seeds.exs

```diff
@@ -9,3 +9,232 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+
+alias Lunchbot.LunchbotData
+
+LunchbotData.create_office(%{
+  name: "Providence",
+  timezone: "GMT-4"
```

Comment:
I recommend using a real timezone here rather than a GMT offset -- because the offset will change for daylight savings time.